force-app/main/default/flowDefinitions/Before_Public_Event_Delete.flowDefinition-meta.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@bmtemakina please remove this metadata XML file from the PR, as we discussed in the daily meeting we don't need flow definition metadata